example/src/main/java/com/example/BUILD.bazel

```diff
@@ -0,0 +1,4 @@
+# ---- BEGIN BZL_GEN_BUILD_GENERATED_CODE ---- no_hash
```

Comment:
we should really generate formatted bazel files... just running buildifier over a big repo is a non-negligible runtime.